docs/_data/nav_tutorial.yml

```diff
@@ -2,89 +2,89 @@
   items:
     - id: tutorial
       title: Before We Start
-      href: /react/tutorial/tutorial.html#before-we-start
```

Comment:
Didn't mean to add this change - it doesn't actually fix the tutorial links. I think it will be helpful though.